src/rdkafka_sasl_scram.c

```diff
@@ -934,7 +934,7 @@ static int unittest_scram_nonce (void) {
         rd_kafka_sasl_scram_generate_nonce(&out2);
         RD_UT_ASSERT(out1.size == 32, "Wrong size %d", (int)out2.size);
 
-        RD_UT_ASSERT(memcmp(out1.ptr, out2.ptr, out1.size == 32) != 0,
```

Comment:
you should give a more strongly typed language a go. can give a recommendation if you want ;-).

Reply:
hehe, you said "go".

Newer clang actually warn about this :+1: